operators/pkg/instctrl/cloudinit.go

```diff
@@ -86,35 +90,34 @@ func (r *InstanceReconciler) EnforceCloudInitSecret(ctx context.Context) error {
 	return nil
 }
 
-// GetWebDavCredentials extracts the credentials (i.e. username and password)
+// Extracts the NFS server name and path for the user's personal NFS volume,
```

Comment:
The comment should start with the name of the function.

Reply:
Changed